client/scripts/controllers/chain/substrate/staking.ts

```diff
@@ -133,21 +147,31 @@ class SubstrateStaking implements StorageModule {
         api.query.staking.currentEra(),
         api.derive.staking.stashes(),
         api.derive.staking.currentPoints(),
-        api.derive.imOnline.receivedHeartbeats()
+        api.derive.imOnline.receivedHeartbeats(),
+        api.derive.staking.electedInfo()
       )),
 
       // fetch balances alongside validators
       flatMap((
-        [api, { nextElected, validators: currentSet }, era, allStashes, queryPoints, imOnline]:
-        [ApiRx, DeriveStakingValidators, EraIndex, AccountId[], EraRewardPoints, Record<string, DeriveHeartbeatAuthor>]
+        [api, { nextElected, validators: currentSet }, era, allStashes,
+          queryPoints, imOnline, electedInfo]:
+        [ApiRx, DeriveStakingValidators, EraIndex, AccountId[],
+        EraRewardPoints, Record<string, DeriveHeartbeatAuthor>, DeriveStakingElected]
       ) => {
         const eraPoints: Record<string, string> = {};
         const entries = [...queryPoints.individual.entries()]
           .map(([accountId, points]) => [accountId.toString(), formatNumber(points)]);
         entries.forEach(([accountId, points]): void => {
           eraPoints[accountId] = points;
         });
+        const commissionInfo: ICommissionInfo = {};
 
+        electedInfo.info.forEach(({ accountId, validatorPrefs }) => {
+          const commissionPer = (validatorPrefs.commission.unwrap() || new BN(0)).toNumber() / 10_000_000;
```

Comment:
`unwrap()` will throw if commission is a `None` option. Use `unwrapOr` instead to provide a default value.

Reply:
Property 'unwrapOr' does not exist on type 'Compact<Perbill>'. So I have added if condition.